bpf/process/generic_calls.h

```diff
@@ -934,13 +934,19 @@ do_action(void *ctx, __u32 i, struct selector_action *actions, bool *post, bool
 		}
 		break;
 	case ACTION_OVERRIDE:
+#if defined(GENERIC_UPROBE) && defined(__TARGET_ARCH_x86)
+		index = actions->act[++i];
+		do_uprobe_override(ctx, index);
+		polacct = POLICY_OVERRIDE;
```

Comment:
Can we respect monitor/enforce mode here as well? Or is there a reason we don't want to?

Reply:
hum, not sure what you mean.. @kkourt